cmd/algod/main.go

```diff
@@ -249,7 +249,23 @@ func main() {
 		log.Fatalf("DefaultDeadlock is somehow not set to an expected value (enable / disable): %s", config.DefaultDeadlock)
 	}
 
-	err = s.Initialize(cfg)
+	var phonebookAddressees []string
+	if peerOverrideArray != nil {
+		phonebookAddressees = peerOverrideArray
+	} else {
+		ex, err := os.Executable()
+		if err != nil {
+			log.Errorf("cannot locate node executable: %s", err)
```

Comment:
if you hit this line you shouldn't keep going with `phonebookDir := filepath.Dir(ex)`, right ?

Reply:
yes. you're right.